cap-primitives/src/yanix/fs/mod.rs

```diff
@@ -56,11 +55,12 @@ pub(crate) use open_options_ext::*;
 pub(crate) use open_unchecked::*;
 pub(crate) use permissions_ext::*;
 pub(crate) use read_dir_inner::*;
-pub(crate) use readlink_one::*;
 pub(crate) use readlink_unchecked::*;
 pub(crate) use remove_dir_all_impl::*;
 pub(crate) use rename_unchecked::*;
 pub(crate) use rmdir_unchecked::*;
 pub(crate) use stat_unchecked::*;
 pub(crate) use symlink_unchecked::*;
 pub(crate) use unlink_unchecked::*;
+
+pub(crate) const MAX_SYMLINK_EXPANSIONS: u8 = 40;
```

Comment:
We can cite https://man7.org/linux/man-pages/man7/path_resolution.7.html as the source for `40` here. This may need to be customized for non-Linux platforms, but I think we can figure that out later.